Jint/Runtime/Intrinsics.cs

```diff
@@ -255,5 +257,8 @@ internal Intrinsics(Engine engine, Realm realm)
 
         public ThrowTypeError ThrowTypeError =>
             _throwTypeError ??= new ThrowTypeError(_engine, _engine.Realm) { _prototype = _engine.Realm.Intrinsics.Function.PrototypeObject };
+        
+        public ErrorConstructor NativeError =>
```

Comment:
I'm not sure if this concept of `NativeError` exists as JS primitive. `TypeError` and friends are `NativeError`s by nature, it's a grouping name for error and their behavior AFAIK.

Reply:
I think you're right, re-reading seems like it's not just a placeholder for "anything-non-js" like I understood it on my first read: https://tc39.es/ecma262/multipage/fundamental-objects.html#sec-nativeerror-object-structure. It's not super clear though, even more confusing when reading TypeError definition: `TypeError is used to indicate an unsuccessful operation when none of the other NativeError objects are an appropriate indication of the failure cause.`

I'll just use Error and move on then ;)

Reply:
Yes, the spec should be always enjoyed with a grain of salt. I generally re-learn things by reading it again after some time. There are some edge-cases where some imagination is required.. 😃